jobs/generation/Utilities.groovy

```diff
@@ -1015,7 +1015,23 @@ class Utilities {
                             }
                         }
                         steps {
-                            batchFile("echo repro build step.")
+                            batchFile("ren 'C:\Jenkins\launch.cmd' 'C:\Jenkins\launch.cmd.disabled'")
+                            def curlCommand = "curl --request POST " +
+                                              "--url \'https://snapshotter.azurewebsites.net/api/snapshot/vm?code=${SNAPSHOT_TOKEN}\' "+
```

Comment:
You'll want the SNAPSHOT_TOKEN to be expanded at runtime (rather than at generation time), so you should use an environment variable: %SNAPSHOT_TOKEN%

It may work the way you have it, but I'm not positive that jenkins variables are replaced in the batchFile step